projects/plugins/jetpack/modules/memberships/class-jetpack-memberships.php

```diff
@@ -743,10 +745,30 @@ public static function user_can_view_post( $post_id = null ) {
 	 * @return bool
 	 */
 	public static function is_enabled_jetpack_recurring_payments() {
-		$api_available = ( ( defined( 'IS_WPCOM' ) && IS_WPCOM ) || Jetpack::is_connection_ready() );
+		$api_available = ( new Host() )->is_wpcom_simple() || Jetpack::is_connection_ready();
 		return $api_available;
 	}
 
+	/**
+	 * Whether to enable the blocks in the editor.
+	 * All Monetize blocks (except Simple Payments) need an active connecting and a user with at least `edit_posts` capability
+	 *
+	 * @return bool
+	 */
+	public static function should_enable_monetize_blocks_in_editor() {
+		if ( ! is_admin() ) {
```

Comment:
I think this is reversed and it should return true when `is_admin()` is true.

Reply:
Okay, I understand this now. `is_admin` refers to an admin page.

Reply:
> I think this is reversed and it should return true when is_admin() is true.

You scared me for a bit: it tested fine so I was wondering what I did wrong here :)